supermarq-benchmarks/supermarq/stabilizers.py

```diff
@@ -133,11 +133,12 @@ def construct_stabilizer(
     pauli_basis = []
     start_idx = 0
     for i, row in enumerate(echelon_matrix.T):
-        if 1 in row[start_idx:]:
+        matrix_row = np.array(row)
```

Comment:
nit (avoids unnecessary copies)
```suggestion
        matrix_row = np.asarray(row)
```